### DISCUSSION TITLE
Does Skip work on Existing Projects?

### DISCUSSION THREAD

**Discussion by Emin-Emini** [skiptools/skip#305] [Q&A] [answered]
Hello,

I’ve been exploring Skip and have a few questions regarding its usage:

Does Skip work with existing projects, or is it limited to creating new projects from scratch?
If it does support existing projects, could you provide guidance on how to integrate Skip into an existing iOS app?

I’d appreciate any clarification or documentation references on these points. Thank you!

**Comment by marcprux** [comment, marked as answer]
Skip works best for new projects. See the coverage of the topic in the documentation at https://skip.tools/docs/gettingstarted/#existing_development

If you have an existing app codebase you would like to use Skip with, we still recommend starting with a Skip-generated project (i.e., the `skip init --appid=…` command described at https://skip.tools/docs/gettingstarted/), and then migrating your existing project into the new one in stages.
